From 2134f5e121f5b1732e4b5de1a030d30b8391ef51 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Wed, 7 Aug 2013 18:52:01 +0800
Subject: [PATCH 404/435] ENGR00274308 Improve some test Makefiles

This patch improves some test Makefiles:
* mxc fb test/mxc ipudev test:
  Pass link options with LIBS variable instead of
  CFLAGS variable.
* mxc v4l2 test:
  Remove -pthread link option for ipu vout test.
  Pass link options with LIBS variable instead of
  CFLAGS variable for pxp vout test.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_fb_test/Makefile     | 2 +-
 test/mxc_ipudev_test/Makefile | 2 +-
 test/mxc_v4l2_test/Makefile   | 4 +---
 3 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/test/mxc_fb_test/Makefile b/test/mxc_fb_test/Makefile
index 5e91ded..18a2ecd 100644
--- a/test/mxc_fb_test/Makefile
+++ b/test/mxc_fb_test/Makefile
@@ -1,6 +1,6 @@
 # list of platforms which did not want this test case
 EXCLUDE_LIST:=
-CFLAGS += -lm
+LIBS += -lm
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-fb.sh $(OBJDIR)/mxc_tve_test.sh $(OBJDIR)/mxc_fb_test.out \
diff --git a/test/mxc_ipudev_test/Makefile b/test/mxc_ipudev_test/Makefile
index 1acc838..7cdc6ee 100644
--- a/test/mxc_ipudev_test/Makefile
+++ b/test/mxc_ipudev_test/Makefile
@@ -2,7 +2,7 @@
 INCLUDE_LIST:=IMX51 IMX53 IMX6Q
 
 SRCS = mxc_ipudev_test.c utils.c
-CFLAGS += -lm -lrt
+LIBS += -lm -lrt
 
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(INCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_ipudev_test.out
diff --git a/test/mxc_v4l2_test/Makefile b/test/mxc_v4l2_test/Makefile
index ff3eca5..194ce25 100644
--- a/test/mxc_v4l2_test/Makefile
+++ b/test/mxc_v4l2_test/Makefile
@@ -6,9 +6,7 @@ NEED_PXP_LIB_LIST:=IMX6S
 # Just use pxp library to get contiguous physical memory for USERPTR testing
 # However, be noticed that this is not the only way.
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(NEED_PXP_LIB_LIST)))
-CFLAGS += -lpthread -lpxp -DGET_CONTI_PHY_MEM_VIA_PXP_LIB
-else
-CFLAGS += -lpthread
+LIBS += -lpthread -lpxp -DGET_CONTI_PHY_MEM_VIA_PXP_LIB
 endif
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
-- 
1.8.0

